power.lib

```diff
@@ -920,7 +920,7 @@ ENDDEF
 #
 # Earth
 #
-DEF ~Earth #PWR 0 0 Y Y 1 F P
+DEF Earth #PWR 0 0 Y Y 1 F P
```

Comment:
Can you explain why the `~ ` got removec

Reply:
That was an old legacy naming convention from, I believe, KiCad v4. Probably this library hasn't been touched in a while so things were updated during the save in the UI.